assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/clients/DataCiteClientSystemTest.java

```diff
@@ -203,20 +205,20 @@ private void stubDeleteMetadataResponse(String expectedPathForDeletingMetadata)
     }
 
     private void verifySetLandingResponse(Doi requestedDoi) {
-        verify(putRequestedFor(urlEqualTo(doiPath))
+        verify(putRequestedFor(urlEqualTo(doiPath + FORWARD_SLASH + requestedDoi.toIdentifier()))
```

Comment:
what is {doiPath}/{requestedDoi.toIdentfier()}? refactor to method?

Reply:
Fixed.